frontend/src/api/api.ts

```diff
@@ -3,6 +3,27 @@ import {CalibrationConfig, CalibrationInfo, ScreeningEvent} from "@/types";
 import {Frame} from "@/camera";
 const API_BASE =
   "https://ixg63w0770.execute-api.ap-southeast-2.amazonaws.com/event";
+const DEVICE_ENDPOINT = (deviceId: string) => `https://3pu8ojk2ej.execute-api.ap-southeast-2.amazonaws.com/default/devices/${deviceId}`
```

Comment:
I'm a fan of semi-colons to end statements myself, it would be good to be consistent with this.